src/main/java/se/sundsvall/esigning/api/SigningResource.java

```diff
@@ -1,16 +1,52 @@
 package se.sundsvall.esigning.api;
 
-import io.swagger.v3.oas.annotations.tags.Tag;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.accepted;
+
+import jakarta.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+
+import se.sundsvall.esigning.api.model.SigningRequest;
+import se.sundsvall.esigning.service.SigningService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
 @RestController
 @Validated
 @RequestMapping("/e-signing")
+@ApiResponses(value = {
+	@ApiResponse(responseCode = "202", description = "Accepted", content = @Content(mediaType = APPLICATION_JSON_VALUE), useReturnTypeSchema = true),
+	@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
+	@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class))),
+	@ApiResponse(responseCode = "503", description = "Service Unavailable", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+})
 @Tag(name = "eSigning", description = "E-signing")
 class SigningResource {
 
-	public SigningResource() {
+	private final SigningService signingService;
+
+	SigningResource(final SigningService signingService) {
+		this.signingService = signingService;
 	}
+
+	@Operation(summary = "Start a signing process")
+	@PostMapping(value = "/start", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_PROBLEM_JSON_VALUE)
+	ResponseEntity<String> startSigningProcess(@Valid @RequestBody final SigningRequest request) {
```

Comment:
Might want to change to some object to be better prepared if we should want to add more parameters in the future. As it is now, it will be a major version when changing to json object (if that should happen) instead of just adding attributes to existing json (which only will be a minor change of API).